workers/unity/Packages/com.improbable.gdk.core/Tests/Editmode/Config/LocatorConfigTests.cs

```diff
@@ -14,6 +14,20 @@ public void UseExternalIp_should_be_true_by_default()
             Assert.IsTrue(config.UseExternalIp);
         }
 
+        [Test]
+        public void ConnectionTimeoutMillis_should_be_60000_by_default()
```

Comment:
This is an oddly specific test. Is this something we want?

Reply:
The main benefit of it was to prevent Rider from complaining "this isn't used anywhere"